test/Microsoft.AspNetCore.Localization.FunctionalTests/LocalizationTest.cs

```diff
@@ -84,7 +84,7 @@ public Task Localization_ResourcesInFolder_ReturnLocalizedValue_AllOS()
                 "http://localhost:5074",
                 "ResourcesInFolder",
                 "fr-FR",
-                "Bonjour from StartupResourcesInFolder Bonjour from Test in resources folder Bonjour from Customer in resources folder Bonjour from Test in resources folder");
```

Comment:
This was a change that we made to all "ResourcesInFolder" tests that was missed in this instance because the test didn't fail.